### PR TITLE
cano: 0-unstable-2024-31-3 -> 0.1.0-alpha

### PR DESCRIPTION
**Pull request by Sigmanificient**
## Description of changes

Move to version `0.1.0-alpha`

## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [X] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by Sigmanificient** [comment]
Result of `nixpkgs-review pr 301955` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cano</li>
  </ul>
</details>

File: pkgs/by-name/ca/cano/package.nix [hunk 1]
```diff
@@ -4,14 +4,14 @@
 , gnumake
 , ncurses
 }:
-stdenv.mkDerivation (finalAttrs: {
+stdenv.mkDerivation (finalAttrs: rec {
```
Comment (Aleksanaa) [inline pkgs/by-name/ca/cano/package.nix:7]:
```suggestion
stdenv.mkDerivation (finalAttrs: {
```

Use `"v${finalAttrs.version}"`

**Comment by Sigmanificient** [comment]
@Aleksanaa not sure why ofborg is failing, may you help investingating it?

**Comment by Aleksanaa** [comment]
@ofborg eval

**Comment by Sigmanificient** [comment]
`error: attribute 'cxxabi' missing`

**Comment by Sigmanificient** [comment]
> `error: attribute 'cxxabi' missing`

Is was a bug introduced in master (see #305610) and was fixed by #305609.

File: pkgs/by-name/ca/cano/package.nix [hunk 2]
```diff
@@ -6,12 +6,12 @@
...
   name = "cano";
-  version = "0-unstable-2024-31-3";
+  version = "0.1.0-alpha";
 
   src = fetchFromGitHub {
     owner = "CobbCoding1";
     repo = "Cano";
-    rev = "6b3488545b4180f20a7fa892fb0ee719e9298ddc";
+    rev = "v${finalAttrs.version}";
     hash = "sha256-qFo0szZVGLUf7c7KdEIofcieWZqtM6kQE6D8afrZ+RU=";
```
Comment (Mindavi) [inline pkgs/by-name/ca/cano/package.nix:15]: Hash needs an update too.

**Review by Aleksanaa — DISMISSED** [review]
That `v0.1.0-alpha` is not `6b3488545b4180f20a7fa892fb0ee719e9298ddc` so you definitely need to update the hash, otherwise you are still building against the old source.

**Pull request merged**